less/variables.less

```diff
@@ -309,66 +309,66 @@
 // Drawer
 // ======
 
-@drawer-color:@primary-color;
-@drawer-text-color:#fff;
+@drawer-color: #424242;
+@drawer-text-color: #fff;
 
-@drawer-icon-color:#fff;
-@drawer-icon-hover-color:darken(@drawer-color, 30%);
+@drawer-icon-color: #fff;
+@drawer-icon-hover-color: darken(@drawer-icon-color, 10%);
 
-@drawer-item-color:#3eb4e4;
-@drawer-item-hover-color:darken(@drawer-item-color,10%);
+@drawer-item-color: #727272;
+@drawer-item-hover-color: darken(@drawer-item-color, 10%);
 
-@drawer-item-text-color:#fff;
-@drawer-item-text-hover-color:#fff;
+@drawer-item-text-color: #fff;
+@drawer-item-text-hover-color: #fff;
 
-@drawer-item-padding-top:10px;
-@drawer-item-padding-right:10px;
-@drawer-item-padding-bottom:10px;
-@drawer-item-padding-left:10px;
+@drawer-item-padding-top: 10px;
+@drawer-item-padding-right: 10px;
+@drawer-item-padding-bottom: 10px;
+@drawer-item-padding-left: 10px;
 
-@drawer-item-padding:@drawer-item-padding-top @drawer-item-padding-right @drawer-item-padding-bottom @drawer-item-padding-left;
+@drawer-item-padding: @drawer-item-padding-top @drawer-item-padding-right @drawer-item-padding-bottom @drawer-item-padding-left;
 
 
 // Notify
 // ======
 
-@notify-color:#3eb4e4;
-@notify-text-color:#fff;
+@notify-color: @primary-color;
+@notify-text-color: #fff;
 
-@notify-icon-color:#fff;
-@notify-icon-hover-color:darken(@notify-color, 30%);
+@notify-icon-color: #fff;
+@notify-icon-hover-color: darken(@notify-color, 30%);
 
-@notify-button-color:#fff;
-@notify-button-hover-color:darken(@notify-color, 30%);
-@notify-button-text-color:#3eb4e4;
-@notify-button-text-hover-color:#fff;
+@notify-button-color: #fff;
+@notify-button-hover-color: darken(@notify-color, 30%);
+@notify-button-text-color: @primary-color;
+@notify-button-text-hover-color: #fff;
 
-@notify-popup-padding-top:30px;
-@notify-popup-padding-right:30px;
-@notify-popup-padding-bottom:30px;
-@notify-popup-padding-left:30px;
+@notify-popup-padding-top: 30px;
+@notify-popup-padding-right: 30px;
+@notify-popup-padding-bottom: 30px;
+@notify-popup-padding-left: 30px;
 
-@notify-popup-padding:@notify-popup-padding-top @notify-popup-padding-right @notify-popup-padding-bottom @notify-popup-padding-left;
+@notify-popup-padding: @notify-popup-padding-top @notify-popup-padding-right @notify-popup-padding-bottom @notify-popup-padding-left;
 
 
 // Fonts
 // =====
 
-@import url(//fonts.googleapis.com/css?family=Open+Sans:300,400,600);
+@import url(https://fonts.googleapis.com/css?family=Roboto:300,400,700);
```

Comment:
Please could you stick to a protocol-less URL like it was before? If you specify the protocol then you will get a secure/insecure content content issue - mainly in IE (which displays a confusing warning) but also I think Chrome too - if the content is run from server that uses a different protocol